ebean-api/src/main/java/io/ebean/bean/EntityBeanIntercept.java

```diff
@@ -360,7 +360,7 @@ public void setForceUpdate(boolean forceUpdate) {
    * Return true if the entity should be updated.
    */
   public boolean isUpdate() {
-    return forceUpdate || state == STATE_LOADED;
+    return forceUpdate || state == STATE_LOADED || state == STATE_REFERENCE;
```

Comment:
In our oppinion this would be the fix.
Why would you want to insert a reference-bean?

Reply:
Yes, I agree with this fix. Thanks